.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jest": true,
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
```

Comment:
If anyone has any other eslint rules that they want to add let me know, these are just common ones that we use on leap.